fastlane_core/lib/fastlane_core/globals.rb

```diff
@@ -0,0 +1,33 @@
+module FastlaneCore
+  class Globals
+    def self.captured_output
+      unless @captured_output
+        @captured_output = ""
+      end
+      @captured_output
```

Comment:
Is there a reason we need to assign `@captured_output = ""` as part of this? Would 

```ruby
def self.captured_output
  @captured_output || ""
end
```

be OK?